docker-compose.yml

```diff
@@ -33,3 +33,74 @@ services:
       - 6832:6832/udp
       - 5778:5778
       - 16686:16686
+  etcd0:
+    image: quay.io/coreos/etcd
+    ports:
+      - "2379:2379"
+    volumes:
+      - etcd0:/etcd_data
+    command:
```

Comment:
Can you put this over a few lines i.e:
```
command: >
  /usr/local/bin/etcd etcd0
  --data-dir /etcd_data
  -advertise-client-urls http://etcd0:2379
  -listen-client-urls http://0.0.0.0:2379
  -initial-advertise-peer-urls http://etcd0:2380
  -listen-peer-urls http://0.0.0.0:2380
  -initial-cluster etcd0=http://etcd0:2380,etcd1=http://etcd1:2380,etcd2=http://etcd2:2380

Reply:
@chriscoffee what's the difference from doing `command: >` and doing it over an array of arguments?